definitions/infra-awslex/dashboard.json

```diff
@@ -0,0 +1,139 @@
+{
+  "name": "Amazon Lex",
+  "description": null,
+  "permissions": "PUBLIC_READ_WRITE",
+  "pages": [
+    {
+      "name": "Amazon Lex",
+      "description": null,
+      "widgets": [
+        {
+          "title": "Audio/Video Logs Delivery Success/Failure",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.area"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [
+                  3635617
+                ],
+                "query": "SELECT count(`aws.lex.ConversationLogsTextDeliveryFailure`)as logsTextFailure, count(`aws.lex.ConversationLogsTextDeliverySuccess`)as logsTextSuccess, count(`aws.lex.ConversationLogsTextDeliverySuccess`)as logsTextSuccess, count(`aws.lex.ConversationLogsAudioDeliverySuccess`)as logsAudioSucces, count(`aws.lex.ConversationLogsAudioDeliveryFailure`)as logsAudioFailure FROM Metric SINCE 1 week AGO TIMESERIES"
```

Comment:
Remove SINCE as it's automatically injected from user active time interval

Reply:
addressed